Source/CryptRaider/Actor/Trigger/Dialog/Pick/GrabberAwareDialogPickStrategy.h

```diff
@@ -0,0 +1,18 @@
+// Fill out your copyright notice in the Description page of Project Settings.
+
+#pragma once
+
+#include "CoreMinimal.h"
+#include "BaseDialogPickStrategy.h"
+#include "ItemAwareDialogPickStrategy.h"
+#include "GrabberAwareDialogPickStrategy.generated.h"
+
+
+UCLASS(ClassGroup=(Custom), meta=(BlueprintSpawnableComponent))
+class CRYPTRAIDER_API UGrabberAwareDialogPickStrategy : public UItemAwareDialogPickStrategy
```

Comment:
We could use new condition based dialog strategy (ConditionDialogStrategy) as parent for ItemAware and GrabberAware...

With virtual condition getter implemented in child strategies)